index.html

```diff
@@ -0,0 +1,112 @@
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        id="header-img"
+      />
+
+      <nav id="nav-bar">
+        <ul>
+          <li><a href="#Features" class="nav-link">Features</a></li>
+          <li><a href="#video" class="nav-link">How it Works</a></li>
+          <li><a href="#Pricing" class="nav-link">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <h1>Handcrafted, home-made masterpieces</h1>
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="Enter your email address"
+        />
+
+        <button id="submit" type="submit">Get Started</button>
+      </form>
+
+      <ul id="Features">
+        <li>
+          <h2>Premium Materials</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium
+            totam praesentium fugit minus itaque, autem iusto fugiat est sed.
+            Rem beatae cupiditate aliquam, aspernatur voluptate itaque omnis
+            odio nisi provident.
+          </p>
+        </li>
+
+        <li>
+          <h2>Fast Shipping</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt
+            magnam modi facere dolor impedit fugiat doloribus eveniet ipsa
+            nostrum! Aperiam vel totam suscipit esse porro eum neque similique
+            alias incidunt?
+          </p>
+        </li>
+
+        <li>
+          <h2>Quality Assurance</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis
+            necessitatibus expedita inventore reprehenderit ipsa eos porro
+            possimus qui voluptas? Incidunt assumenda similique magnam, saepe
+            quos autem facere cumque voluptas dolor.
+          </p>
+        </li>
+      </ul>
+
+      <iframe
+        id="video"
+        width="560"
+        height="315"
+        src="https://www.youtube.com/embed/tgB1wUcmbbw"
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+      ></iframe>
+      <div id="Pricing"></div>
+      <div class="contain">
+        <h1>
+          <div>
+            <h3></h3>
+            <p></p>
+          </div>
+        </h1>
+      </div>
+
+      <div class="contain">
+        <h1>
+          <div>
+            <h3></h3>
+            <p></p>
+          </div>
+        </h1>
+      </div>
+
+      <div class="contain">
+        <h1>
+          <div>
+            <h3></h3>
+            <p></p>
+          </div>
+        </h1>
```

Comment:
Please remove unused code or add corresponding content

Reply:
Please remove unused code or add corresponding content to make your code cleaner and readable